packages-web/maps/src/Maps.editorConfig.ts

```diff
@@ -134,13 +135,40 @@ export function check(values: MapsPreviewProps): Problem[] {
         }
     });
 
-    values.dynamicMarkers.forEach(marker => {
+    values.dynamicMarkers.forEach((marker, index) => {
         if (!marker.markersDS) {
             errors.push({
                 property: "dynamicMarkers.markersDS",
                 message: "A data source should be selected in order to retrieve a list of markers"
             });
         }
+        if (marker.locationType === "address") {
+            if (!marker.address) {
+                errors.push({
+                    property: "dynamicMarkers.address",
+                    message: "A dynamic marker requires an address"
+                });
+            }
+        } else {
+            if (!marker.latitude) {
+                errors.push({
+                    property: "dynamicMarkers.latitude",
+                    message: "A dynamic marker requires latitude"
+                });
+            }
+            if (!marker.longitude) {
+                errors.push({
+                    property: "dynamicMarkers.longitude",
+                    message: "A dynamic marker requires longitude"
+                });
+            }
+        }
+        if (values.advanced && marker.markerStyleDynamic === "image" && !marker.customMarkerDynamic) {
+            errors.push({
+                property: "dynamicMarkers.customMarkerDynamic",
+                message: `Custom marker image is required when shape is 'image' for list at position ${index + 1}`
```

Comment:
This one is the only where we mention `index`. Isn't it a bit misaligned since errors are very similar?

Reply:
We are gonna do another interaction with Maria about the consistency errors next week, so as we talked i will gonna ignore this comment for now